.github/workflows/ci.yaml

```diff
@@ -46,7 +46,6 @@ jobs:
     env:
       ## US preset
```

Comment:
Please keep client ID here, it's part of presets

Reply:
moved